shuffle/move/examples/e2e/message.test.ts

```diff
@@ -27,25 +27,35 @@ Deno.test("Ability to set message", async () => {
 
 Deno.test("Ability to set NFTs", async () => {
   // Initialize nft_collection resource for both sender and receiver
-  let senderInitializeTxn = await main.initializeNFTScriptFunction();
-  senderInitializeTxn = await devapi.waitForTransactionCompletion(
+  let senderInitializeTxn = await main.initializeNFTScriptFunction(
+    "TestNFT",
+    context.defaultUserContext,
+    context.defaultUserContext.address,
+  );
+  senderInitializeTxn = await devapi.waitForTransaction(
     senderInitializeTxn.hash,
   );
   const secondUserContext = context.UserContext.fromEnv("test");
   let receiverInitializeTxn = await main.initializeNFTScriptFunction(
+    "TestNFT",
     secondUserContext,
+    context.defaultUserContext.address,
   );
-  receiverInitializeTxn = await devapi.waitForTransactionCompletion(
+  receiverInitializeTxn = await devapi.waitForTransaction(
     receiverInitializeTxn.hash,
   );
   assert(senderInitializeTxn.success);
   assert(receiverInitializeTxn.success);
 
   // Mint TestNFT into sender address
   const contentUri = "https://placekitten.com/200/300";
-
-  let txn = await main.createTestNFTScriptFunction(contentUri);
-  txn = await devapi.waitForTransactionCompletion(txn.hash);
+  let txn = await main.createTestNFTScriptFunction(
+    contentUri,
+    "TestNFT",
+    context.defaultUserContext,
+    context.defaultUserContext.address,
```

Comment:
Same here. These last two lines are optional and with default are redundant to the defaults. Is for illustrative purposes?